pkg/apis/migration/v1alpha1/v1alpha1_suite_test.go

```diff
@@ -40,16 +42,24 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		golog.Fatal(err)
 	}
-
-	if cfg, err = t.Start(); err != nil {
-		golog.Fatal(err)
+	setupControlPlane := os.Getenv("SETUP_CONTROL_PLANE")
```

Comment:
This is a flag @alaypatel07 has installed to allow for running of tests on MacOS environments where the control plane cannot be setup properly. This is purely optional and doesn't represent any change needed for non-MacOS devs.